routes/sessions.js

```diff
@@ -0,0 +1,62 @@
+var Bcrypt = require('bcrypt');
+var Joi = require('joi');
+var Auth = require('./auth');
+
+exports.register = function(server,options,next) {
+  server.route([
+    {
+      method: 'POST',
+      path: '/sessions',
+      config: {
+        handler: function(request,reply) {
+          var db = request.server.plugins['hapi-mongodb'].db;
+          var user = request.payload.user;
+          db.collection('users').findOne({username : user.username},function(err,userMongo) {
+            if(err) {reply('Internal Mongo Error ', err);}
```

Comment:
if you use `if(err) {return reply('Internal Mongo Error ', err);}`, then you don't need an `else` statement